packages/runtime-core/src/vnode.ts

```diff
@@ -801,5 +800,14 @@ export function mergeProps(...args: (Data & VNodeProps)[]) {
       }
     }
   }
+  for (let i = 0; i < args.length; i++) {
+    const arg = args[i]
+    if (isArray(arg)) {
+      // support v-bind="[propsA, propsB]"
```

Comment:
These changes will increase runtime overhead.
I think transform `v-bind="[propsA, propsB]"` to `mergeProps(propsA, propsB)` will be better.

Reply:
> These changes will increase runtime overhead.
> I think transform `v-bind="[propsA, propsB]"` to `mergeProps(propsA, propsB)` will be better.

Just use `v-bind="{...propsA, ...propsB}"`
no need to create this stupid runtime checking and conversion.